client/verta/verta/dataset/_dataset.py

```diff
@@ -305,29 +307,29 @@ class Component(object):
     """
     def __init__(
             self,
-            path, size=None, last_modified=None,
+            path, base_path=None,
```

Comment:
Note that you just broke api compatibility since the order of arguments is part of the API, right? I don't think you need to. Not a big deal in this case, but just pointing out to help you practice.